h5p.classes.php

```diff
@@ -705,16 +717,17 @@ public function isValidPackage($skipContent = FALSE, $upgradeOnly = FALSE) {
     else {
       $this->h5pF->setErrorMessage($this->h5pF->t('The file you uploaded is not a valid HTML5 Package (We are unable to unzip it)'));
       H5PCore::deleteFileTree($tmpDir);
-      return;
+      return FALSE;
     }
     unlink($tmpPath);
 
     // Process content and libraries
     $valid = TRUE;
     $libraries = array();
     $files = scandir($tmpDir);
-    $mainH5pData;
-    $libraryJsonData;
```

Comment:
You cannot declare variables like this in PHP, it may be forgiving and run since you are technically declaring it during association, but this is a red level warning